ci-operator/step-registry/mirror-images/by-oc-mirror/mirror-images-by-oc-mirror-commands.sh

```diff
@@ -113,8 +126,14 @@ cp -rf "${new_pull_secret}" "${XDG_RUNTIME_DIR}/containers/auth.json"
 
 unset REGISTRY_AUTH_PREFERENCE
 
+mirrorCmd="${oc_mirror_bin} -c ${image_set_config} docker://${target_release_image_repo} --dest-tls-verify=false --v2 --workspace file://${oc_mirror_dir}"
+
+# To workaround OCPBUGS-56009
+if ! isPreVersion "4.19"; then
```

Comment:
I just found `OCPBUGS-56009` already backport to 4.19, sound like the bug is already fixed, if that, I guess we do not need this workaround any more.

Reply:
https://github.com/openshift/oc-mirror/pull/1147/files added the option --ignore-release-signature to fix this bug

Reply:
If that, need to judge the target payload is singed or not, if signed, we should NOT append the option blindly.

Reply:
How to check the target payload is signed or not?

Reply:
You can check `check_signed` function in `cucushift-upgrade-toimage` to migrate it to this step.